src/base/bittorrent/torrentinfo.h

```diff
@@ -29,12 +29,13 @@
 #ifndef BITTORRENT_TORRENTINFO_H
 #define BITTORRENT_TORRENTINFO_H
 
-#include <QCoreApplication>
 #include <QtGlobal>
```

Comment:
In this case, QtGlobal should move down with QCoreApplication, there is no need to put it here.